vortex-array/src/array/chunked/compute/filter.rs

```diff
@@ -57,11 +57,12 @@ fn slices_to_mask(slices: &[(usize, usize)], len: usize) -> FilterMask {
     let n_trailing_false = len - pos;
     buffer.append_n(n_trailing_false, false);
 
-    FilterMask::from(BoolArray::from(buffer.finish()))
+    FilterMask::from(buffer.finish())
 }
 
 /// Filter the chunks using slice ranges.
-fn filter_slices(array: &ChunkedArray, mask: &FilterMask) -> VortexResult<Vec<ArrayData>> {
+#[allow(deprecated)]
```

Comment:
what is deprecated...?

Reply:
Calling iter_indices or iter_slices directly on a FilterMask. There are a few filter implementations that need fixing up in FLUP PRs